helm-chart/kuberay-operator/README.md

```diff
@@ -52,8 +52,8 @@ To list the `my-release` deployment:
 
 ```sh
 helm ls
-# NAME                    NAMESPACE       REVISION        UPDATED                                   STATUS          CHART                   # APP VERSION
-# kuberay-operator        default         1               2022-12-02 02:13:37.514445313 +0000 UTC   deployed        kuberay-operator-0.6.0  1.0
+# NAME                    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                           APP VERSION
```

Comment:
This is exactly what I observed when running `helm ls`.  The field `APP VERSION` is empty since no `appVersion` in `Chart.yaml`.